pkg/flameql/flameql.go

```diff
@@ -21,6 +21,7 @@ type TagMatcher struct {
 
 type Op int
 
+//revive:disable:var-naming Those names mimics constants names
```

Comment:
Actually, I used this notation for no good reason. Perhaps, it would be better to refactor names (e.g:
`OpEqual`, `OpEqualRegex`, etc),